src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -6,13 +6,41 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private Car car;
+
+    public Driver(Long id, String name, String licenseNumber, Car car) {
```

Comment:
Remove the constructor with the Car parameter, as the relationship between Driver and Car should not be in the Driver class.